src/pages/custom-fields/custom-fields.html

```diff
@@ -0,0 +1,27 @@
+<ion-header>
```

Comment:
Would it make sense to abstract this header component eventually?
I see this being a nice abstraction, but I of course don't know if it would cover all cases:
```
<stockpile-header title="Custom Fields" onAdd="onAdd()" />
```
The `onAdd` prop would be optional. When `undefined`, the component would hide the plus button; when set, it would show the plus button and call the provided function.

This is me spitballing. :) It would only make sense to abstract it if it is a significant source of duplication.

Reply:
Yes I think so! I was actually thinking the same thing today while looking at the code.

It will most likely be done with #415